modules/custom-status/views/configure.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+defined( 'ABSPATH' ) || exit();
+
+global $edit_flow;
+
+?>
+
+<div id="col-right">
+	<div class="col-wrap">
+		<?php $custom_status_list_table->display(); ?>
+		<?php wp_nonce_field( 'custom-status-sortable', 'custom-status-sortable' ); ?>
+		<p class="description" style="padding-top:10px;"><?php esc_html_e( 'Deleting a post status will assign all posts to the default post status.', 'edit-flow' ); ?></p>
+	</div>
+</div>
+
+<div id="col-left">
+	<div class="col-wrap">
+		<div class="form-wrap">
+			<h3 class="nav-tab-wrapper">
+				<?php $add_new_nav_class = empty( $action ) ? 'nav-tab-active' : ''; ?>
+				<a href="<?php echo esc_url( $this->get_link() ); ?>" class="nav-tab <?php echo esc_attr( $add_new_nav_class ); ?>"><?php esc_html_e( 'Add New', 'edit-flow' ); ?></a>
+				<?php $options_nav_class = 'change-options' === $action ? 'nav-tab-active' : ''; ?>
+				<a href="<?php echo esc_url( $this->get_link( array( 'action' => 'change-options' ) ) ); ?>" class="nav-tab <?php echo esc_attr( $options_nav_class ); ?>"><?php esc_html_e( 'Options', 'edit-flow' ); ?></a>
+			</h3>
+
+			<?php if ( 'change-options' === $action ) { ?>
+			<form class="basic-settings" action="<?php echo esc_url( $this->get_link( array( 'action' => 'change-options' ) ) ); ?>" method="post">
+				<?php settings_fields( $this->module->options_group_name ); ?>
+				<?php do_settings_sections( $this->module->options_group_name ); ?>
+				<?php echo '<input id="edit_flow_module_name" name="edit_flow_module_name" type="hidden" value="' . esc_attr( $this->module->name ) . '" />'; ?>
```

Comment:
I think we can avoid `echo`ing out the entire input field if we instead just echo out the module name.
Like this:
```suggestion
				<input id="edit_flow_module_name" name="edit_flow_module_name" type="hidden" value="<?php echo esc_attr( $this->module->name ) ?>" />
```